CHANGELOG.md

```diff
@@ -5,6 +5,129 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 8.12.0
+
+_Sep 25, 2025_
+
+We'd like to extend a big thank you to the 15 contributors who made this release possible. Here are some highlights ✨:
+
+- ⌨️ Charts keyboard navigation
+- ⚡️ Charts: Add new `BatchScatter` component that can optionally replace `Scatter` for large datasets
```

Comment:
I'd rewrite it as such:

```suggestion
- ⚡️ Charts: Add new `renderer="svg-batch"` prop to Scatter charts that provides improved performance for large datasets
```

The `BatchScatter` component is an implementation detail. Users will enable this feature by setting the `renderer="svg-batch"` prop on `ScatterChart`